### ISSUE TITLE
TypeError in Global Error Trapper

### ISSUE THREAD

**Issue by TrentPierce** [thkruz/keeptrack.space#956]
#### User Description
Type what you were trying to do here...

I enabled debris screening and then clicked the "Screen for debris" button. This tab opened and I got a NULL TypeError. This is likely because I left the Satellite SCC# empty, but this page was unclear on how it works. Here is a screenshot...

![debris screening](https://github.com/user-attachments/assets/10c9ee98-9ad0-485c-a9ce-dc4671da0c23)

#### Version
10.1.0 - September 29, 2024
#### Error Title
TypeError
#### Error Message
r is null
#### Stack
84681/onFormSubmit/c<@https://app.keeptrack.space/js/856.131053e7e7529c62c836.js:629:1501
onFormSubmit@https://app.keeptrack.space/js/856.131053e7e7529c62c836.js:629:1432
84681/addJs/cb/</<@https://app.keeptrack.space/js/856.131053e7e7529c62c836.js:629:296
64272/l/<@https://app.keeptrack.space/js/main.1b9159195eca89309707.js:2:1397765

**Comment by thkruz** [comment]
Thanks for the picture - they really help. There was some old code used for enabling/disabling that button and it incorrectly allows you to have the menu open without first selecting a satellite. That will be fixed with the next PR.

You can read more about that plugin here: https://docs.keeptrack.space/plugins/debris-screening/

tl;dr version: NASA searches around the ISS using that 0.5 x 25 x 25 km box. They propagate the satellite along its orbit and continuously scan for anything that comes within that box. After chatting with someone from their office, I made that plugin to help them explain it to the public. It will work with any satellite and you can even do an actual search, but its meant for the ISS and the search is admittedly slow.
